<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>301</width>
      <height>321</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="btn_add">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>70</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_5">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>130</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_divide">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>/</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_9">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>190</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_multiply">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>190</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>*</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_8">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>190</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_6">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_0">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>250</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_subtract">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>130</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_decimal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>.</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_parentheses_right">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_parentheses_left">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_clear">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>clear</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_answer">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>10</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>ans</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_equals">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>250</y>
       <width>51</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="list_output">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>301</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textbox_input">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Input Number:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
